src/Mod/Sketcher/Gui/DrawSketchHandlerLine.h

```diff
@@ -306,11 +306,17 @@ void DSHLineController::configureToolWidget()
     onViewParameters[OnViewParameter::Second]->setLabelType(Gui::SoDatumLabel::DISTANCEY);
 
     if (handler->constructionMethod() == ConstructionMethod::OnePointLengthAngle) {
-        onViewParameters[OnViewParameter::Fourth]->setLabelType(Gui::SoDatumLabel::ANGLE);
+        onViewParameters[OnViewParameter::Fourth]->setLabelType(
+            Gui::SoDatumLabel::ANGLE,
+            Gui::EditableDatumLabel::Function::Dimensioning);

```

Comment:
This is probably not working in 'dimension ovp only' mode. Because here `OnViewParameter::Third` was not set because it is using the default 'DISTANCE' type. But if you don't set it now, then it will default at Positioning function.
So you need to add : 

```
        onViewParameters[OnViewParameter::Third]->setLabelType(
            Gui::SoDatumLabel::DISTANCE,
            Gui::EditableDatumLabel::Function::Dimensioning);
```

Reply:
I did a quick test and I had the impression it worked. Perhaps I am mistaken. Just give it a try when you cherry-pick those commits...